Documentation/ceph-teardown.md

```diff
@@ -32,10 +32,24 @@ kubectl delete -f csi/cephfs/kube-registry.yaml
 kubectl delete storageclass csi-cephfs
 ```
 
-## Delete the CephCluster CRD
-
 After those block and file resources have been cleaned up, you can then delete your Rook cluster. This is important to delete **before removing the Rook operator and agent or else resources may not be cleaned up properly**.
 
+## Automatic Cleanup
+
+Edit the `CephCluster` and add the `cleanupPolicy`
+
+WARNING: DATA WILL BE PERMANENTLY DELETED AFTER DELETING THE `CephCluster` CR WITH `cleanupPolicy`.
+
+```console
+kubectl -n rook-ceph patch cephcluster rook-ceph --type merge -p '{"spec":{"cleanupPolicy":{"confirmation":"yes-really-destroy-data"}}}'
+```
+
+Once the cleanup policy is enabled, any new configuration changes in the CephCluster will be blocked. Nothing will happen unitl the deletion of the CR is requested, so this `cleanupPolicy` change can still be reverted.
+
+Checkout more details about the `cleanupPolicy` [here](https://rook.io/docs/rook/master/ceph-cluster-crd.html#cleanup-policy)
```

Comment:
Actually just refer to the .md files in the links, and the build process replaces with the correct html and branch link
```suggestion
Checkout more details about the `cleanupPolicy` [here](ceph-cluster-crd.md#cleanup-policy)
```